jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -292,6 +300,17 @@ public Context getServletContext()
         return _scontext;
     }
 
+    @ManagedAttribute("The format of IPv6 addresses return from the servlet API")
+    public IPV6Format getIpv6Format()
```

Comment:
Since the class is called `IPV6Format` (capital `P`), this method should also be `getIPv6Format()` with capital `P`.